Assets/Scripts/FengGameManagerMKII.cs

```diff
@@ -3546,8 +3556,7 @@ public void spawnPlayerAtRPC(float posX, float posY, float posZ, PhotonMessageIn
             component.enabled = true;
             GameObject.Find("MainCamera").GetComponent<IN_GAME_MAIN_CAMERA>().setHUDposition();
             GameObject.Find("MainCamera").GetComponent<SpectatorMovement>().disable = true;
-            //TODO MouseLook
-            //GameObject.Find("MainCamera").GetComponent<MouseLook>().disable = true;
+            GameObject.Find("MainCamera").GetComponent<Assets.Scripts.UI.Camera.MouseLook>().disable = true;
```

Comment:
Import the namespace, and use MouseLook instead of Assets.Scripts.UI.Camera.MouseLook